### PR TITLE
Adds Select component

### PR DESCRIPTION
**Pull request by asbjornh**
Select component that can be styled without breaking touch and keyboard interaction

### CODE REVIEW

File: components/select/select.jsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import cn from "classnames";
+
+// NOTE: This is a naiive implementation of 'lodash/get'. Replave it if you have 'lodash' as a dependency
```
Comment (Reasonable-Solutions) [inline components/select/select.jsx]: `replave`!

Comment (Reasonable-Solutions) [reply components/select/select.jsx]: `naiive`!

Comment (asbjornh) [reply components/select/select.jsx]: 😅

Comment (asbjornh) [reply components/select/select.jsx]: Fixed the typos and a couple of other ones

**Pull request merged**